examples/unix/main.c

```diff
@@ -273,23 +302,46 @@ main(int argc, char *argv[])
         return 0;
     }
 
-    // send assets from account 0 to account 1
-    char *notes = (char *) "Alice sent 1 Algo to Bob";
-    err_code =
-        vertices_transaction_pay_new(alice_account_handle,
-                                     (char *) bob_account.public_key,
-                                     AMOUNT_SENT,
-                                     notes);
-    VTC_ASSERT(err_code);
+    switch (run_tx)
+    {
+        case PAY_TX:
+        {
+            // send assets from account 0 to account 1
+            char *notes = (char *) "Alice sent 1 Algo to Bob";
+            err_code =
+                vertices_transaction_pay_new(alice_account_handle,
+                                             (char *) bob_account.public_key,
+                                             AMOUNT_SENT,
+                                             notes);
+            VTC_ASSERT(err_code);
+        }
+            break;
+
+        case APP_CALL_TX:
+        {
+            // send application call
+            app_values_t kv = {0};
+            kv.count = 1;
+            kv.values[0].type = VALUE_TYPE_INTEGER;
+            kv.values[0].value_uint = 20;
+
+            err_code = vertices_transaction_app_call(alice_account_handle, 16037129, &kv);
```

Comment:
Same remark as in https://github.com/vertices-network/c-vertices-sdk/pull/5/files#r656474309